### PR TITLE
Replaced Bugsnag with Crashlytics

### PR DESCRIPTION
**Pull request by MalcolmMcFly**
What's this do?
This replaces Bugsnag with Crashlytics

Why are we doing this? (w/ JIRA link if applicable)
No Jira task for this feature.

How should this be tested? / Do these changes have associated tests?
Simply running the app and forcing a crash will show in Firebase Console

Dependencies for merging? Releasing to production?
One thing to note, I did have to add dexOptions { javaMaxHeapSize "4g" } to get the build.gradle to build after adding Firebase.

Has the application documentation been updated for these changes?
Yes, I've removed the mentions of Bugsnag from the Readme and project licenses.

Did someone actually run this code to verify it works?

### CODE REVIEW

**Comment by io7m** [comment]
Looks good, but I think you forgot to update the Travis CI script to copy the `google-services.json` into place.

File: app/build.gradle [hunk 1]
```diff
@@ -63,6 +63,10 @@ android {
   lintOptions {
     checkReleaseBuilds false
   }
+
+  dexOptions {
+    javaMaxHeapSize "4g"
```
Comment (twaddington) [inline app/build.gradle]: I'm pretty sure this option is no longer used and may simply be ignored. I'm having trouble finding where I saw that though.

Comment (twaddington) [reply app/build.gradle]: Does this fix some build issue that you were seeing?

**Review by twaddington — APPROVED** [review]
Looks good!

**Review by twaddington — APPROVED** [review]

**Pull request merged**